src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -712,7 +712,7 @@ export default defineComponent({
       }
 
       this.description = this.data.description
-      this.isLive = this.data.liveNow || this.data.lengthSeconds === 'undefined'
+      this.isLive = this.data.liveNow || this.data.lengthSeconds === 'undefined' || Number.isNaN(this.data.lengthSeconds)
```

Comment:
Please improve the parsing (you already started but just didn't go all the way), instead of adding a workaround at display time.

Parsing changes that are needed:
- Setting the `liveNow` property
- Making sure that the lengthSeconds property is an empty string for live videos.